med_io/write_tf_record.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
 import os
 
-def write_tfrecord(data, path):
+def write_tfrecord(data, validation_for_cancer, path):
```

Comment:
backward comp. broken -> do not change `write_tfrecord` interface -> split to separate write function

Reply:
Another function added in write_tf_record.py file. The function "write_tfrecord_melanoma" was added and adapted in preprocessing